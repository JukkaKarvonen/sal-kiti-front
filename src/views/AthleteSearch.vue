<template>
  <div class="search">
    <AthleteSearch />
  </div>
</template>

<script>
/**
 * Athlete search view
 */
import AthleteSearch from "@/components/AthleteSearch.vue";

export default {
  name: "AthleteSearchView",
  components: {
    AthleteSearch
  },
  created() {
    document.title = this.$t("title.search") + " | " + this.$t("title.prefix");
  }
};
</script>
