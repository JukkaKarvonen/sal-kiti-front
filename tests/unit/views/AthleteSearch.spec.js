import { shallowMount, createLocalVue } from "@vue/test-utils";
import AthleteSearch from "@/views/AthleteSearch.vue";

import BootstrapVue from "bootstrap-vue";

const localVue = createLocalVue();
localVue.use(BootstrapVue);

describe("AthleteSearch.vue", () => {
  it("renders content", () => {
    const wrapper = shallowMount(AthleteSearch, {
      localVue
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
