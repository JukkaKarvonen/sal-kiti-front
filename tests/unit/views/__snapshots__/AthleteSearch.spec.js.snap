// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AthleteSearch.vue renders content 1`] = `
<div
  class="search"
>
  <athletesearch-stub />
</div>
`;
